crates/forge/src/runner.rs

```diff
@@ -165,14 +165,53 @@ impl<'a> ContractRunner<'a> {
             traces.extend(setup_traces.map(|traces| (TraceKind::Setup, traces)));
             logs.extend(setup_logs);
 
-            TestSetup { address, logs, traces, labeled_addresses, reason, coverage }
+            TestSetup {
+                address,
+                logs,
+                traces,
+                labeled_addresses,
+                reason,
+                coverage,
+                fuzz_fixtures: self.fuzz_fixtures(address, fixtures),
+            }
         } else {
-            TestSetup::success(address, logs, traces, Default::default(), None)
+            TestSetup::success(
+                address,
+                logs,
+                traces,
+                Default::default(),
+                None,
+                self.fuzz_fixtures(address, fixtures),
+            )
         };
 
         Ok(setup)
     }
 
+    /// Collect fixtures from test contract. Fixtures are functions prefixed with `fixture_` key
+    /// and followed by the name of the parameter.
+    ///
+    /// For example:
+    /// `fixture_test() returns (address[] memory)` function
+    /// define an array of addresses to be used for fuzzed `test` named parameter in scope of the
+    /// current test.
+    fn fuzz_fixtures(&mut self, address: Address, fixtures: &InlineFixturesConfig) -> FuzzFixtures {
```

Comment:
Shouldn't this be more strict considering that we no more require `fixture_` prefix?

Reply:
yeah, updated comment to reflect also the possibility of fixtures from storage arrays